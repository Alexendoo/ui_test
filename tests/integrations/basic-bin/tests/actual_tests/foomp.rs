use basic_bin::add;
//~^ ERROR: extern location for basic_bin is of an unknown type
//~| ERROR: file name should be lib*.rlib or lib*.so
//~| ERROR: can't find crate for `basic_bin`

fn main() {
    add("42", 3);
}
