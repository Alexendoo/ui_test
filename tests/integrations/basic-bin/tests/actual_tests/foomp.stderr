error: extern location for basic_bin is of an unknown type: $DIR/tests/integrations/basic-bin/../../../target/$TRIPLE/debug/basic_bin
 --> $DIR/foomp.rs:1:5
  |
1 | use basic_bin::add;
  |     ^^^^^^^^^

error: file name should be
 --> $DIR/foomp.rs:1:5
  |
1 | use basic_bin::add;
  |     ^^^^^^^^^

error[E0463]: can't find crate for `basic_bin`
 --> $DIR/foomp.rs:1:5
  |
1 | use basic_bin::add;
  |     ^^^^^^^^^ can't find crate

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0463`.
